docs/en/migrations/snowflake.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: Snowflake
+sidebar_position: 20
+slug: /en/migrations/snowflake
+description: Migrating from Snowflake to ClickHouse
+keywords: [migrate, migration, migrating, data, etl, elt, snowflake]
+---
+
+# Migrating from Snowflake to ClickHouse
+
+This guide shows how to migrate data from Snowflake to ClickHouse.
+
+Migrating data between Snowflake and ClickHouse requires the use of an object store, such as S3, as an intermediate storage for transfer. The migration process also relies on using the commands `COPY INTO` from Snowflake and `INSERT INTO SELECT` of ClickHouse.
+
+## 1. Exporting data from Snowflake
+
+<img src={require('./images/migrate_snowflake_clickhouse.png').default} class="image" alt="Migrating from Snowflake to ClickHouse" style={{width: '600px', marginBottom: '20px', textAlign: 'left'}}/>
+
+Exporting data from Snowflake requires the use of an external stage, as shown in the diagram above.
+
+Let's say we want to export a Snowflake table with the following schema:
+
+```sql
+CREATE TABLE MYDATASET (
+   timestamp TIMESTAMP,
+   some_text varchar,
+   some_file OBJECT,
+   complex_data VARIANT,
+) DATA_RETENTION_TIME_IN_DAYS = 0;
+```
+
+To move this table's data to a ClickHouse database, we first need to copy this data to an external stage. When copying data, we recommend Parquet as the intermittent format as it allows type information to be shared, preserves precision, compresses well, and natively supports nested structures common in analytics.
+
+In the example below, we create a named file format in Snowflake to represent Parquet and the desired file options. We then specify which bucket will contain our copied dataset. Finally, we copy the dataset to the bucket.
+
+```sql
+CREATE FILE FORMAT my_parquet_format TYPE = parquet;
+
+-- Create the external stage that specifies the S3 bucket to copy into
+CREATE OR REPLACE STAGE external_stage 
+URL='s3://mybucket/mydataset'
+CREDENTIALS=(AWS_KEY_ID='<key>' AWS_SECRET_KEY='<secret>')
+FILE_FORMAT = my_parquet_format;
+
+-- Apply "mydataset" prefix to all files and specify a max file size of 150mb
+-- The `header=true` parameter is required to get column names
+COPY INTO @external_stage/mydataset from mydataset max_file_size=157286400 header=true;
+```
+
+For a dataset around 5TB of data with a maximum file size of 150MB, and using a 2X-Large Snowflake warehouse located in the same AWS `us-east-1` region, copying data to the S3 bucket will take around 30 minutes.
```

Comment:
Not verified this, i assume you lifted from the blog and sounds roughly right. Users should only pay for warehouse compute if its in the same region also.